eclair-core/src/main/scala/fr/acinq/eclair/db/PaymentsDb.scala

```diff
@@ -105,7 +105,7 @@ case object PaymentType {
  * @param createdAt       absolute time in milli-seconds since UNIX epoch when the payment request was generated.
  * @param status          current status of the payment.
  */
-case class IncomingPayment(paymentRequest: PaymentRequest,
+case class IncomingPayment(paymentRequest: Bolt11Invoice,
```

Comment:
I think it's worth renaming fields that are currently called `paymentRequest` and use `invoice` everywhere, what do you think? As long as we're touching this code everywhere, better do it once for good?

Reply:
Big renaming done